.github/workflows/pre-commit.yaml

```
@@ -0,0 +1,15 @@
+name: pre-commit
+
+on: [ push, pull_request ]
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)